src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -118,6 +118,46 @@ SpatialPooler::SpatialPooler()
   version_ = 2;
 }
 
+SpatialPooler::SpatialPooler(vector<UInt> inputDimensions,
+                                  vector<UInt> columnDimensions,
```

Comment:
This spacing is still off. The "vector" in this line should be aligned with the "vector" in the previous and following lines for this construction should match as well.

And for the initialize call below, don't align it with this call since it is a completely separate statement.

Alignment for added construction in `hpp` file is good